docs/user-guide/tips-tricks-and-anti-patterns.ipynb

```diff
@@ -413,6 +420,224 @@
     "stacked = sc.concat([da, da], dim='z')\n",
     "sc.show(stacked)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Improving performance: Allocators and HugePages\n",
+    "\n",
+    "### Overview\n",
+    "\n",
+    "Scipp frequently needs to allocate or deallocate large chunks of memory.\n",
+    "In many cases this is a major contribution to the (lack of) performance of Scipp (or any other) applications.\n",
+    "Aside from minimizing the number of allocations (which is not always easy or possible), there are two simple ways that have each shown to independently yield up to 5% or 10% performance improvements for certain workloads."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Using the TBB allocation proxy\n",
+    "\n",
+    "Scipp uses TBB for parallelization.\n",
+    "TBB also provides allocators, but Scipp is not using them by default.\n",
+    "It is possible to automatically do so by preloading `libtbbmalloc_proxy.so`.\n",
+    "For more context refer to the [oneTBB documentation](https://oneapi-src.github.io/oneTBB/main/tbb_userguide/automatically-replacing-malloc.html).\n",
+    "\n",
+    "<div class=\"alert alert-info\">\n",
+    "\n",
+    "**Note**\n",
+    "    \n",
+    "Make sure to install Scipp via `conda` since Scipp's PyPI package does not ship the required `libtbbmalloc_proxy.so`.\n",
+    "    \n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Python\n",
+    "\n",
+    "Use\n",
+    "\n",
+    "```console\n",
+    "$ LD_PRELOAD=libtbbmalloc_proxy.so.2 python\n",
+    "```\n",
+    "\n",
+    "to launch Python or\n",
+    "\n",
+    "```console\n",
+    "$ LD_PRELOAD=libtbbmalloc_proxy.so.2 python myapp.py\n",
+    "```\n",
+    "\n",
+    "to run a Python script.\n",
+    "Note that the `.2` suffix may be different depending on the actual version of TBB."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Jupyter\n",
+    "\n",
+    "Environment variables can be configured for a specific Jupyter kernel.\n",
+    "First, obtain a list of kernels:\n",
+    "\n",
+    "```console\n",
+    "$ jupyter kernelspec list\n",
+    "```\n",
+    "\n",
+    "The list may include, e.g., `/home/<username>/mambaforge/envs/myenv/share/jupyter/kernels/python3`.\n",
+    "Consider making a copy of this folder, e.g., to `/home/<username>/mambaforge/envs/myenv/share/jupyter/kernels/python3-tbbmalloc_proxy`.\n",
```

Comment:
This is Linux-specific. 
That being said, does the preload mechanism work on MacOS and Windows?

Reply:
According to the docs it is possible on Windows, but I do not know how to preload. On MacOS the lib is available, but the docs to not mention support... maybe it is implied in "Linux"?

Reply:
I added a visible note at the top of the section, marking it is Linux-only.